content/appendix/audit_reports.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Audit Reports
+weight: 4
+dashboardWeight: 0.2
+dashboardState: reliable
+dashboardAudit: n/a
+---
+
+# Audit Reports
+
+Security is a critical component in ensuring Filecoin can fulfill its mission to be the storage network for humanity. In addition to robust secure development processes, trainings, theory audits, and investing in external security research, the Filecoin project has engaged reputable third party auditing specialists to ensure that the theory behind the protocol and its implementation delivers the intended value, enabling Filecoin to be a safe and secure network. This section covers a selection of audit reports that have been published on Filecoin's theory and implementation.
+
+## GossipSub
+
+### 2020-06-03: GossipSub Design and Implementation
+
+Audit report: [GossipSub v1.1 Protocol Design + Implementation](https://gateway.ipfs.io/ipfs/QmWR376YyuyLewZDzaTHXGZr7quL5LB13HRFnNdSJ3CyXu/Least%20Authority%20-%20Gossipsub%20v1.1%20Final%20Audit%20Report%20%28v2%29.pdf)
+
+This audit focused specifically on GossipSub as a pubsub protocol built on libp2p, looking at v1.1 which includes a peer scoring layer to mitigate certain types of attacks that could compromise a network. The audit covered the [spec](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub), [go-libp2p-pubsub](https://github.com/libp2p/go-libp2p-pubsub) and [gossipsub-hardening](https://github.com/libp2p/gossipsub-hardening/). The report found 4 issues, primarily in the Peer Scoring that was introduced in v1.1, and includes additional suggestions. All the issues raised in the report have been resolved, and additional details are available in the report linked above.
+
+### 2020-04-18: GossipSub Evaluation
+
+Audit report: [GossipSub-v1.1 Evaluation Report](https://gateway.ipfs.io/ipfs/QmRAFP5DBnvNjdYSbWhEhVRJJDFCLpPyvew5GwCCB4VxM4)
```

Comment:
I believe @dkkapur wanted to link to Least Authority Security Audit instead.

Reply:
Yeah the LA report is included as well. I think I will change the "audit report" to just "report" and since this has evaluation in its title, hopefully should be OK?